autogpt_platform/backend/backend/executor/scheduler.py

```
@@ -217,18 +377,38 @@ async def _execute_graph(**kwargs):
                 f"Graph execution {graph_exec.id} took {elapsed:.2f}s to create/publish - "
                 f"this is unusually slow and may indicate resource contention"
             )
+        return graph_exec.id
     except GraphNotFoundError as e:
+        # Not retryable — mark missed without billing and unschedule
+        try:
+            await ScheduleOccurrence.prisma().update(where={"id": occurrence.id}, data={"status": "missed"})
+        except Exception:
+            pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)